server/core/src/https/v1.rs

```diff
@@ -636,15 +642,15 @@ pub async fn person_id_delete(
 pub async fn service_account_get(
     State(state): State<ServerState>,
     Extension(kopid): Extension<KOpId>,
-) -> Result<Json<Vec<ProtoEntry>>, WebError> {
+) -> Result<Json<Vec<ProtoEntryReal>>, WebError> {
     let filter = filter_all!(f_eq(Attribute::Class, EntryClass::ServiceAccount.into()));
     json_rest_event_get(state, None, filter, kopid).await
 }
 
 #[utoipa::path(
     post,
     path = "/v1/service_account",
-    request_body=Json, // TODO ProtoEntry can't be serialized, so we need to do this manually
+    request_body=ProtoEntry, // TODO ProtoEntryReal can't be serialized, so we need to do this manually
```

Comment:
Most of this is just replacing `Json<Thing>` to `Thing` because the openapi doesn't really care if it's json at this level.